src/components/Header/utils/encoreCatalogLogOutTimer.ts

```diff
@@ -1,6 +1,7 @@
 import Cookies from "js-cookie";
 
-export const encoreLogOutURL =
+// TODO
+export const catalogLogOutURL =
```

Comment:
@nonword I can't easily find the logout url. Do you know what it is?

Reply:
`https://[dev-]login.nypl.org/auth/logout?redirect_uri=RETURN_URL`